README.md

```diff
@@ -8,35 +6,56 @@ Terraform module to create a Function App which automates certificate management
 
 | Name | Version |
 |------|---------|
-| terraform | >= 0.14.6 |
-| azurerm | =2.58.0 |
+| <a name="requirement_terraform"></a> [terraform](#requirement\_terraform) | >= 0.15.1 |
```

Comment:
these links don't seem to do anything?

Reply:
There is bug in the code. Links aren't valid ones. I am using the latest version of terraform-docs.